tests/test-helper.ts

```diff
@@ -5,6 +5,7 @@ import { setRunOptions } from 'ember-a11y-testing/test-support';
 import { start } from 'ember-qunit';
 import * as QUnit from 'qunit';
 import { setup } from 'qunit-dom';
+import 'qunit-dom';
```

Comment:
`qunit-dom` suggested (at the time of writing) adding this import statement so that `assert` takes the correct type (i.e. `assert.dom` becomes well-defined).